frappe/website/doctype/blog_post/blog_post.json

```diff
@@ -197,6 +198,12 @@
    "fieldname": "disable_feedback",
    "fieldtype": "Check",
    "label": "Disable Feedback"
+  },
+  {
+   "default": "1",
+   "fieldname": "enable_email_notification",
+   "fieldtype": "Check",
+   "label": "Enable Email Notification"
```

Comment:
@shariquerik Better to have a description below that check. Currently, it is not very clear that the "Email Notification" is related to comments and feedback.

Reply:
Added👍

Reply:
If it looks good will update the docs